example_data/epilepsy_bundle.json

```diff
@@ -0,0 +1,656 @@
+              "coding": [
+                {
+                  "system": "http://loinc.org",
+                  "code": "53034-0",
+                  "display": "inheritance"
+                }
+              ]
+            },
+            "valueCodeableConcept": {
+              "coding": [
+                {
+                  "system": "http://loinc.org",
+                  "code": "LA6705-0",
+                  "display": "AR"
+                }
+              ]
+            }
+          },
+          {
+            "code": {
+              "coding": [
+                {
+                  "system": "http://hl7.org/geneNames",
+                  "code": "gene-names",
+                  "display": "Gene Name"
+                }
+              ]
+            },
+            "valueString": " BRAT1"
+          },
+          {
+            "code": {
+              "coding": [
+                {
+                  "system": "http://hl7.org/evidence",
+                  "code": "evidence",
+                  "display": "Evidence for classification of variant"
+                }
+              ]
+            },
+            "valueString": "predicted null variant where loss of function is known disease mechanism (PVS1_very strong)."
+          },
+          {
+            "code": {
+              "coding": [
+                {
+                  "system": "http://hl7.org/mutation",
+                  "code": "confirmed-mutation",
+                  "display": "confirmed-mutation"
+                }
+              ]
+            },
+            "valueBoolean": false
+          }
+        ],
+        "note": [
+          {
+            "authorString": "comments",
+            "text": "This variant occurs in a recessive gene that has been 100% sequenced and no second variant identified."
+          }
+        ]
+      },
+      "request": {
+        "method": "POST",
+        "url": "Observation"
+      }
+    },
+    {
+      "fullUrl": "Patient/43a92b3e-d757-11ec-9d64-0242ac120002",
+      "resource": {
+        "id": "43a92b3e-d757-11ec-9d64-0242ac120002",
+        "resourceType": "Patient",
+        "active": true,
+        "name": [
+          {
+            "family": "Island",
+            "given": [
+              "London"
+            ]
+          }
+        ],
+        "gender": "male",
+        "birthDate": "2016-03-11",
+        "identifier": [
+          {
+            "value": "1146253",
+            "use": "official"
+          }
+        ]
+      },
+      "request": {
+        "method": "POST",
+        "url": "Patient"
+      }
+    },
+    {
+      "fullUrl": "Specimen/43a96978-d757-11ec-9d64-0242ac120002",
+      "resource": {
+        "resourceType": "Specimen",
+        "id": "43a96978-d757-11ec-9d64-0242ac120002",
+        "receivedTime": "2020-05-22T14:18:00",
+        "collection": {
+          "collectedDateTime": "2020-05-22T12:26:00",
+          "collector": {
+            "display": "Anita Sfagliato"
+          }
```

Comment:
This is also in the report and can't be removed.

Reply:
I think my concern here is that we don't have a record of who physically collected the sample - we know the clinical scientists who analysed the sequencing but nothing else - if that's not the case then it would be good to have some examples of reports here!

Reply:
confirmed remove collector as we don't know this